src/components/PDOKAutoSuggest/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AutoSuggest from 'components/AutoSuggest';
+
+const serviceParams = [
+  ['fq', 'bron:BAG'],
+  ['fq', 'type:adres'],
+  // ['fl', '*'], // undocumented; requests all available field values from the API
+  ['q', ''],
+];
+const serviceURL = 'https://geodata.nationaalgeoregister.nl/locatieserver/v3/suggest?';
+const numOptionsDeterminer = data => data?.response?.docs?.length || 0;
+const formatResponse = ({ response }) => response.docs.map(({ id, weergavenaam }) => ({ id, value: weergavenaam }));
+
+/**
+ * Geocoder component that specifically uses the PDOK location service to request information from
+ *
+ * @see {@link https://www.pdok.nl/restful-api/-/article/pdok-locatieserver#/paths/~1suggest/get}
+ */
+const PDOKAutoSuggest = ({ gemeentenaam, onSelect }) => {
```

Comment:
I see no capability to set a value from props as discussed. Would you please add that?

Reply:
I know; forgot about that. Just added it.